technologies/README.md

```diff
@@ -1,26 +1,42 @@
 
-For a more complete collection of tools, refer to the [Awesome Robotic Deliberation](https://github.com/ros-wg-delib/awesome-ros-deliberation) repository.
+## Introduction
 
----
+This folder contains all the technology-specific implementations of ROS 2 Deliberation technologies.
+Here you will find the tools and libraries that you can learn about in this repository.
+
+For a more complete collection of tools and resources for deliberation in ROS2, please refer to the [Awesome Robotic Deliberation](https://github.com/ros-wg-delib/awesome-ros-deliberation) repository.
 
 ## Skill Abstractions
 
 While is it possible to directly program our robots to perform complex, long-horizon tasks with plain code, it can be useful to think at higher levels of abstraction.
 There are several formalisms that can help you assemble robust, reactive behaviors for your robots.
 Arguably, the two most popular ones are **finite-state machines (FSMs)** and **behavior trees (BTs)**.
 
-In this workshop, you will be using the following software tools:
+Importantly, all these tools (and many others in the ROS ecosystem) provide a programmatic interface for you to write basic skills in code that leverages ROS, and a graphical interface to assemble the skills together into full behaviors and monitor their execution.
 
-* [BehaviorTree.CPP](https://github.com/BehaviorTree/BehaviorTree.CPP) - A C++ based library for behavior trees.
-* [ros_bt_py](https://github.com/fzi-forschungszentrum-informatik/ros2_ros_bt_py) - A Python based library for behavior trees.
-* [FlexBE](https://github.com/FlexBE) - A Python based library for hierarchical finite-state machines.
+With this repository, you will be able to use the following technologies:
 
-Importantly, all these tools (and many others in the ROS ecosystem) provide a programmatic interface for you to write basic skills in code that leverages ROS, and a graphical interface to assemble the skills together into full behaviors and monitor their execution.
+- [BehaviorTree.CPP](./BehaviorTree.CPP/README.md)
+  - A C++ based library for behavior trees.
+  - [github repo](https://github.com/BehaviorTree/BehaviorTree.CPP)
+  - [documentation](https://www.behaviortree.dev/)
+  - [**To start using it to solve the problems, refer to the BehaviorTree.CPP subfolder**](./BehaviorTree.CPP/README.md).
+
+- [ros_bt_py](./ros_bt_py/README.md)
+  - A Python based library for behavior trees.
+  - [github repo](https://github.com/fzi-forschungszentrum-informatik/ros2_ros_bt_py)
+  - [**To start using it to solve the problems, refer to the ros_bt_py subfolder**](./ros_bt_py/README.md).
 
----
+- [FlexBE](./FlexBE/README.md)
+  - A Python based library for hierarchical finite-state machines.
+  - [github org](https://github.com/FlexBE)
+  - [**To start using it to solve the problems, refer to the the FlexBE subfolder**](./FlexBE/README.md).
```

Comment:
for the last sub-bullets of each, should the URL only be placed on the `technology subfolder` part, instead of the full sentence?

Reply:
My thought was to highlight it to differentiate it from the other (external) links as a primary navigation path. But the bold formatting also does that, to be fair.